docs/reference/bpmn-processes/multi-instance/multi-instance.md

```diff
@@ -91,6 +91,16 @@ source: =x
 target: output
 ```
 
+## Completion condition
+
+A multi-instance element is completed when all child instances are completed. However, it is possible to specify an FEEL-expression that is evaluated each time an instance of a multi-instance element completes:
+
+- If it cannot be evaluated as a `boolean`, an incident is raised
+- If it evaluates to `false`, the normal completion behavior of the multi-instance body is used
+- If it evaluates to `true`, any still active child instances of the multi-instance are terminated and the multi-instance element is completed
```

Comment:
🔧 I'd prefer to take the exclusive gateway conditions as example and describe the different scenarios in text, rather than in a list of options. That can makes this description less pseudo-code and more descriptive of its function.